src/firmware_update/mod.rs

```diff
@@ -31,6 +31,7 @@ use tokio::sync::mpsc::Sender;
 pub static SUPPORTED_MSD_DEVICES: Lazy<Vec<(u16, u16)>> = Lazy::new(|| {
     let mut supported = Vec::<(u16, u16)>::new();
     supported.push(rpi_fwupdate::VID_PID);
+    supported.push(rockusb_fwudate::Chip::RK3566.vid_pid());
```

Comment:
This is a "hack" to make the various API endpoints work for this device. Technically it's not fully supported, given I've never done a full flash via the API, so it kind of lies right now.